setup.py

```diff
@@ -463,7 +478,10 @@ def setup_pytorch_extension() -> setuptools.Extension:
     ]
 
     # Compiler flags
-    cxx_flags = ["-O3"]
+    cxx_flags = [
+        "-O3",
+        "-Wno-return-local-addr",
```

Comment:
`transformer_engine/pytorch/csrc/comm_gemm_overlap.h::271` raises this warning. I suppressed it for now because changing the function interface risks breaking user code.